src/averageAges.js

```diff
@@ -38,6 +70,19 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+
+  const women = people.filter(person => person.sex === 'f')
```

Comment:
```suggestion

function isWoman(person) {
  const WOMAN = 'f';
  
  return person.sex === WOMAN;
}

const women = people.filter(isWoman)
```

Reply:
Consider avoiding repeating code for sex checking

Reply:
I've added a function to get women from the list.